package.json

```diff
@@ -18,13 +18,13 @@
     "build": "npm-run-all build:*",
     "build:font-awesome": "node scripts/build-fontawesome.js",
     "build:webpack": "webpack",
-    "watch": "webpack -w",
+    "watch": "webpack --watch",
     "test": "npm-run-all -c test:* lint",
     "test:mocha": "mocha",
     "lint": "npm-run-all -c lint:*",
     "lint:js": "eslint .",
     "lint:css": "stylelint \"**/*.css\"",
-    "prepublish": "npm run build"
+    "prepublishOnly": "NODE_ENV=production npm run build"
```

Comment:
What @YaManicKill said. You can still run `npm run build` on Windows, and if someone ever needs to deploy from Windows, which should only happen in case something goes wrong on Travis CI, one can always change `prepublishOnly` so a Windows-friendly command temporarily.